lib/src/platform_info/platform_info_io.dart

```diff
@@ -0,0 +1,23 @@
+import 'dart:io' show Platform;
+
+import 'platform_info_interface.dart';
+
+class PlatformInfoImpl implements PlatformInfoInterface {
```

Comment:
it's kind of weird to have two different implementation with the same name. Even though I get why it's done like this. Maybe you can set different names and get the same result with the `as` keyword ?

Reply:
Unfortunately it's not possible, for the conditional import to work the 2 implementations have to be strictly identical.